src/app/shared/pipes/string-cutter.pipe.ts

```diff
@@ -0,0 +1,13 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'stringCutter'
+})
+export class StringCutterPipe implements PipeTransform {
+  readonly maxLength = 20;
```

Comment:
please make this params as pipe argument.
And` readonly maxLength = 20` leave as default